bin/util.js

```diff
@@ -20,6 +20,22 @@ module.exports = {
             return num.valueOf();
         },
 
+        /**
+         * Helper to coerce number like strings into integers.
+         * If supplied argument is infinite, return positive infinity
+         * Perform safety checks, and return the result.
+         *
+         * @param {String} arg - The stringified number argument or infinite.
+         * @returns {Number} - The supplied argument, casted to an integer or positive infinity.
+         */
+        integerOrInfinity: (arg) => {
```

Comment:
How does Newman handle other properties that can have infinity? Any pattern to draw from there?